### PR TITLE
reduce some test sizes in Miri

### PR DESCRIPTION
**Pull request by RalfJung**

### CODE REVIEW

**Comment by RalfJung** [comment]
r? @Centril 
Since you just reviewed the other one. ;)

File: src/liballoc/tests/btree/set.rs [hunk 1]
```diff
@@ -166,6 +176,17 @@ fn test_is_subset() {
     assert_eq!(is_subset(&[1, 2], &[1]), false);
     assert_eq!(is_subset(&[1, 2], &[1, 2]), true);
     assert_eq!(is_subset(&[1, 2], &[2, 3]), false);
+    assert_eq!(is_subset(&[-5, 11, 22, 33, 40, 42],
+                         &[-12, -5, 14, 23, 11, 34, 22, 38, 33, 42, 39, 40]),
+               true);
+    assert_eq!(is_subset(&[-5, 11, 22, 33, 40, 42],
+                         &[-12, -5, 14, 23, 34, 38, 22, 11]),
```
Comment (Centril) [inline src/liballoc/tests/btree/set.rs]: Formatting is a bit odd there... but rustfmt is going to fix that anyways... :)

Comment (RalfJung) [reply src/liballoc/tests/btree/set.rs]: I tried to follow the style of other tests in this file. But yes, it is odd.

**Pull request merged**